tests/upgrades/test_satellitesync.py

```diff
@@ -14,138 +14,120 @@
 
 :Upstream: No
 """
-from fauxfactory import gen_string
 from nailgun import entities
 from upgrade_tests import post_upgrade
 from upgrade_tests import pre_upgrade
-from upgrade_tests.helpers.scenarios import create_dict
-from upgrade_tests.helpers.scenarios import get_entity_data
 
 from robottelo import ssh
 from robottelo.cli.contentview import ContentView
 from robottelo.cli.package import Package
-from robottelo.test import CLITestCase
 
 
-class scenario_preversion_cv_exports_imports(CLITestCase):
-    """Test Content-view created before upgrade can be exported and imported after upgrade
+@pre_upgrade
+def test_pre_version_cv_export_import(request):
+    """Before Upgrade, Create the content view and publish, and promote it.
 
-    Test Steps:
+    :id: preupgrade-f19e4928-94db-4df6-8ce8-b5e4afe34258
 
-    1. Before upgrade, Create a ContentView
-    2. Publish and promote the CV
-    3. Upgrade the satellite
-    4. Post upgrade, Export and Import the Content Views version created before upgrade
+    :steps:
 
-    :expectedresults: Content-view created before upgrade should be exported and imported
-        after upgrade
-    """
+    1. Create a ContentView
+    2. Publish and promote the Content View
+    3. Check the package count of promoted content view.
 
+    :expectedresults: Before the upgrade, Content view published and promoted, and  package count
+    should be greater than 0.
+    """
+    org = entities.Organization(name=request.node.name + "_org").create()
+    product = entities.Product(organization=org, name=request.node.name + "_prod").create()
+    repo = entities.Repository(
+        product=product,
+        name=request.node.name + "_repo",
+        mirror_on_sync=False,
+        download_policy='immediate',
+    ).create()
+    repo.sync()
+    cv = entities.ContentView(name=request.node.name + "_cv", organization=org).create()
+    cv.repository = [repo]
+    cv.update(['repository'])
+    cv.publish()
+    cv = cv.read()
+    assert cv.version[0].read().package_count > 0
+
+
+@post_upgrade(depend_on=test_pre_version_cv_export_import)
+def test_post_version_cv_export_import(request):
+    """After upgrade, content view version import and export works on the existing content
+     view(that we created before the upgrade).
+
+    :id: postupgrade-f19e4928-94db-4df6-8ce8-b5e4afe34258
+
+    :steps:
+        1: Export the existing content-view version.
+        2: Import the existing content-view version.
+        3: Delete the imported and exported content-vew, product, repo and organization.
+
+    :expectedresults: After upgrade,
+        1: Content view created before upgrade should be imported and exported successfully.
+        2: Imported and Exported content view should be deleted successfully
+    """
+    pre_test_name = [
+        mark.kwargs['depend_on'].__name__
+        for mark in request.node.own_markers
+        if 'depend_on' in mark.kwargs
+    ][0]
```

Comment:
Fixed.

Reply:
I didnt get what you have fixed there.

Reply:
I have added fixtures in pytest_fixtures/upgrade_fixtures.py, please refer to PR: https://github.com/SatelliteQE/robottelo/pull/8186

Reply:
@devendra104 Do u have a plan to merge this PR once #8186 is merged since you would want to rebase and use that fixture in this PR ?

If thats the plan move this PR to draft or mark it as Do Not Merge.

Reply:
@jyejare I think we are good with both, Can you please review and merge #8186 first?

Reply:
Merged #8186 as all my comments were satisfied.